### ISSUE TITLE
Example of an issue and associated data change

### ISSUE THREAD

**Issue by SOLV-Code** [SOLV-Code/UltraLite-Fisheries-Data-System#11]
Just for illustration, imagine one of the reviewers identified a potential error in [this data file](https://github.com/SOLV-Code/UltraLite-Fisheries-Data-System/blob/main/data/Profiles/YkCk_BorderMR_Data.csv).

The concern can be described in an issue like this, and once it is submitted, a unique identifier is assigned.

**Comment by SOLV-Code** [comment]
In this case the issue was assigned **#11**. If you this this label, including the **#** in any other issues submission, or in the git commit, it will be included as a hyperlink back to this page.

**Comment by SOLV-Code** [comment]
For this example, two numbers in [this data file](https://github.com/SOLV-Code/UltraLite-Fisheries-Data-System/blob/main/data/Profiles/YkCk_BorderMR_Data.csv) were changed, and then the change was committed with the following steps in [Git Bash command line](https://www.geeksforgeeks.org/working-on-git-bash/):

```
git add .
git status
git commit -m "BorderMR data fix as per #11"
git push origin main
```

**Comment by SOLV-Code** [comment]
Once the change is confirmed, you can close this issue thread. It will no long show up in the list of open issues, but it will still be stored and can be found by keyword search (e.g., *BorderMR*). In order to find things later, make sure the commit messages are meaningful!
